charts/secrets-store-csi-driver/values.yaml

```diff
@@ -8,3 +8,7 @@ logLevel:
 ## Install Default RBAC roles and bindings
 rbac:
   install: true
+
+providers:
```

Comment:
let's make sure at least one provider is specified here, otherwise we will end with `MountVolume.SetUp failed for volume "providers-dir" : hostPath type check failed: /etc/kubernetes/secrets-store-csi-providers is not a directory`

Reply:
Instead, should we do `DirectoryOrCreate` in hostPath?

Reply:
we could but the driver should have at least one provider

Reply:
I think we can enable at least one provider in the helm chart once we cut the new release for csi-driver and update the helm chart with the image. In this PR just updating the csi-driver chart is fine with the old image because most of the changes are mounts and hostPath? WDYT?

Reply:
SGTM